filters/filter_24_QuickFixes/template.txt

```diff
@@ -0,0 +1,7 @@
+! Homepage: https://github.com/AdguardTeam/AdGuardFilters
+! License: https://github.com/AdguardTeam/AdguardFilters/blob/master/LICENSE
+!
+!-------------------------------------------------------------------------------!
+!-------------- Fixing filtration problems -------------------------------------!
+!-------------------------------------------------------------------------------!
+@include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/QuickFixesFilter/sections/quickfixes.txt"
```

Comment:
Should we also include `general_js_api.txt`, similar like it's done in other our filters, or in this case it's not necessary?

Reply:
No. At least now.